.shellspec

```diff
@@ -0,0 +1,12 @@
+--require spec_helper
+
+## Default kcov (coverage) options
+# --kcov-options "--include-path=. --path-strip-level=1"
```

Comment:
So these kcov options are all placeholders since they're commented out?

Reply:
Its default from shellspec --init along with spec/spec_helper.sh. Only useful if we want to include coverage which we can at some point. For now I just used all the defaults.